<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>950</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>1200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Signal Analyzer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(47, 47, 47);
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(213, 213, 213);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="spectralBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>561</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Spectral Analysis</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>511</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>FFT window length:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="windowType">
        <item>
         <property name="text">
          <string>hamming</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>hann</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>triang</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>blackman</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Type:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Window Length:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Number MFCCs:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>DCT type:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QSpinBox" name="dctSpin">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>3</number>
        </property>
        <property name="value">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Window Type:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="fftSpin">
        <property name="maximum">
         <number>2048</number>
        </property>
        <property name="value">
         <number>512</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>lifter</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QSpinBox" name="nMFCCSpin">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Window Hop:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="lifterSpin">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>2.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="analysisType">
        <item>
         <property name="text">
          <string>DFT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>FFT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>STFT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MFCC</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QDoubleSpinBox" name="winHopSpin">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>30.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="winlenSpin">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>60.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="analyseButton">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>200</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Analyse</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="analysisBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>1151</width>
      <height>611</height>
     </rect>
    </property>
    <property name="title">
     <string>Analysis</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="speechBox">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>0</y>
      <width>121</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Speech Analysis</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>105</width>
       <height>181</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="transcribeButton">
        <property name="text">
         <string>Transcribe</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pitchButton">
        <property name="text">
         <string>Pitch detection</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="recognitionButton">
        <property name="text">
         <string>Speaker Library</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="digitButton">
        <property name="text">
         <string>Digit Recognition</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="filterBox">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>0</y>
      <width>451</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Filter</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>421</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Filter type:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="filterTypeBox">
          <item>
           <property name="text">
            <string>BUTTERWORTH</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CHEBYSHEW1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CHEBYSHEW2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ELLIPTIC</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Filtering type:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="filteringTypeBox">
          <item>
           <property name="text">
            <string>HPF</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>LPF</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>BPF</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>f0:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="f0SpinBox">
          <property name="maximum">
           <number>50000</number>
          </property>
          <property name="value">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>f1:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="f1SpinBox">
          <property name="maximum">
           <number>50000</number>
          </property>
          <property name="value">
           <number>1000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Order:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="orderSpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>20</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>rp:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="rpSpinBox">
          <property name="maximum">
           <double>10.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>5.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>190</y>
       <width>284</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="filtResponse">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>Show filter frequency response</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="filterButton">
        <property name="text">
         <string>Filter</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>870</y>
      <width>160</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="settingsBox">
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closingButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuExport_as">
     <property name="title">
      <string>Export as</string>
     </property>
     <addaction name="actionJPEG"/>
     <addaction name="actionPNG"/>
     <addaction name="actionWAV"/>
    </widget>
    <addaction name="menuExport_as"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionJPEG">
   <property name="text">
    <string>JPEG</string>
   </property>
  </action>
  <action name="actionPNG">
   <property name="text">
    <string>PNG</string>
   </property>
  </action>
  <action name="actionWAV">
   <property name="text">
    <string>WAV</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
